subprojects/language-scala/src/integTest/groovy/org/gradle/language/scala/PlayCompilationFixture.groovy

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.language.scala
+
+import org.gradle.integtests.fixtures.RepoScriptBlockUtil
+import org.gradle.language.scala.internal.toolchain.DefaultScalaToolProvider
+import org.gradle.test.fixtures.file.TestFile
+
+import static org.gradle.integtests.fixtures.RepoScriptBlockUtil.jcenterRepository
+
+class PlayCompilationFixture {
```

Comment:
❌ This test fixture doesn't seem to be very idiomatic for play. IOW, I think this should be a somewhat minimal play application with the corresponding classes.
It would probably also be nice to see the caching working under the presence of routes and twirl templates.

Reply:
I have added classes for minimal play application. Please let me know if more changes are required